index.d.ts

```diff
@@ -34,6 +34,7 @@ export interface TreeProps {
   nodes: Node[];
   onChange: (nodes: Node[]) => void;
   children: (props: RendererProps) => JSX.Element
+  nodeMarginLeft: number;
```

Comment:
This is an optional param, the correct definition is `nodeMarginLeft?: number;`

Reply:
Oh dang, sorry about that! I've not yet had a chance to really dive into Typescript, so I'm not super familiar with it.

Reply:
Don't worry too much 😄